tfsdk/attribute_test.go

```diff
@@ -803,6 +803,39 @@ func TestAttributeValidate(t *testing.T) {
 			},
 			resp: ValidateAttributeResponse{},
 		},
+		"deprecation-message": {
```

Comment:
Would love to see a test for when the value is:

* present
* absent
* unknown